src/canary-create-account.js

```diff
@@ -32,18 +41,52 @@ const basicCustomEntryPoint = async () => {
 
   const bucketName = await getParameter("bucket");
   const fireDrill = await getParameter("fire-drill");
-  const testServicesApiHostname = await getParameter("test-services-api-hostname");
+  const testServicesApiHostname = await getParameter(
+    "test-services-api-hostname"
+  );
   const testServicesApiKey = await getParameter("test-services-api-key");
-  const syntheticsUserDeletePath = await getParameter("synthetics-user-delete-path");
+  const syntheticsUserDeletePath = await getParameter(
+    "synthetics-user-delete-path"
+  );
+
   const email = await getParameter("username");
   const phoneNumber = await getParameter("phone");
   const password = crypto.randomBytes(20).toString("base64url");
+  const clientBaseUrl = await getParameter("client-base-url");
+  const clientPrivateKey = await getParameter("client-private-key");
+  const clientId = await getParameter("client-id");
+  const issuerBaseURL = await getParameter("issuer-base-url");
 
   if (fireDrill === "1") {
     log.info("Fire Drill! Smoke test will fail.");
     throw "Smoke Test failed due to Fire Drill";
   }
 
+  log.info("Starting Client");
+
+  const port = 3031;
+  app.use(
+    auth({
+      issuerBaseURL: issuerBaseURL,
+      baseURL: clientBaseUrl,
+      clientID: clientId,
+      secret: crypto.randomBytes(20).toString("base64url"),
+      clientAuthMethod: "private_key_jwt",
+      clientAssertionSigningKey: clientPrivateKey,
+      idTokenSigningAlg: "ES256",
+      authRequired: true,
+      authorizationParams: {
+        response_type: "code",
+        scope: "openid email phone",
+      },
+    })
+  );
+  app.get("/", requiresAuth(), async (req, res) => {
```

Comment:
Since you've said `authRequired: true` in the authentication configuration, you don't need the call to `requiresAuth()`.  This also means you don't need to unpack `requiresAuth` on line 12.  Probably only saves you a few bytes of memory, but might be worth doing.

Reply:
Removed